documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -24,26 +25,60 @@ A synopsis of the ``radix_sort`` and ``radix_sort_by_key`` functions is provided
    sycl::event
    radix_sort (sycl::queue q, Range&& r, KernelParam param); // (1)
 
-   template <bool IsAscending = true,  std::uint8_t RadixBits = 8,
+   template <bool IsAscending = true, std::uint8_t RadixBits = 8,
              typename KernelParam, typename Iterator>
    sycl::event
    radix_sort (sycl::queue q, Iterator first, Iterator last,
                KernelParam param); // (2)
 
+   // Sort a single sequence out of place
+
+   template <bool IsAscending = true, std::uint8_t RadixBits = 8,
+             typename KernelParam, typename Range1, typename Range2>
+   sycl::event
+   radix_sort(sycl::queue q, Range1&& r, Range2&& r_out,
+              KernelParam param) // (3)
```

Comment:
Let's align the calls to follow that suggestion: https://github.com/oneapi-src/oneDPL/pull/1328#discussion_r1466631788.

```suggestion
   radix_sort (sycl::queue q, Range1&& r, Range2&& r_out,
               KernelParam param) // (3)
```

Reply:
done.